community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -289,70 +346,101 @@ object projects:
   lazy val sconfig = SbtCommunityProject(
     project       = "sconfig",
     sbtTestCommand   = "sconfigJVM/test",
+    // sbtDocCommand = "sconfigJVM/doc", // Fails with:
+    // Problem parsing sconfig/sharedScala3/src/main/scala/org/ekrich/config/ConfigSyntax.scala:[73..92..1340], documentation may not be generated.
+    // scala.MatchError: ValDef(JSON,TypeTree[TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class ekrich)),module config),class ConfigSyntax)],Apply(Ident($new),List(Literal(Constant(0)), Literal(Constant(JSON))))) (of class dotty.tools.dotc.ast.Trees$ValDef)
   )
 
   lazy val zio = SbtCommunityProject(
     project = "zio",
     sbtTestCommand = "testJVMDotty",
+    // sbtDocCommand  = forceDoc("coreJVM"),
+    // Fails on tasty unpickling https://github.com/lampepfl/dotty/issues/10499
   )
 
   lazy val munit = SbtCommunityProject(
-    project = "munit",
-    sbtTestCommand  = "testsJVM/test;testsJS/test;",
+    project          = "munit",
+    sbtTestCommand   = "testsJVM/test",
     // Hardcode the version to avoid having to deal with something set by sbt-dynver
     sbtPublishCommand   = s"""set every version := "${Versions.munit}"; munitJVM/publishLocal; munitJS/publishLocal; munitScalacheckJVM/publishLocal; munitScalacheckJS/publishLocal; junit/publishLocal""",
-    dependencies = List(scalacheck)
+    sbtDocCommand   = "munitJVM/doc",
   )
```

Comment:
Why are the tests and dependencies for `munit` changed here?

Reply:
good catch, I resolve confilcts many times for this PR and I must make a mistake.